README.md

```diff
@@ -28,7 +28,32 @@ Install Requirements - Start HTTP Server. After running the following playbook y
 $ ansible-playbook setup.yml --ask-sudo-pass
 ```
 
-**Replace the Variables in `vars/atomic.yml` with your httpserver IP Address and OSTree name.**
+**Replace the Variables in `vars/atomic.yml` with your httpserver IP Address, OSTree name, Basehost.**
+
+**If you wish to use CentOS Atomic modify the variables accordingly.**
+
+```
+---
+# Variables for Atomic host
+atomicname: my-atomic
+basehost: fedora-atomic/25/x86_64/docker-host
+httpserver: 192.168.122.1
+```
+
+**Add the additional Packages you would like to have in the OSTree in `vars/buildrepo.yml`.**
+
+**If you wish to use CentOS Atomic modify the variables accordingly.**
+
+```
+---
+repo: https://pagure.io/fedora-atomic.git
+version: f25
+repodir: fedora-atomic
+localcompose: customized-atomic-docker-host.json
```

Comment:
Wasn't sure what to name it. I have named the json file cloned from source as `sourcecompose`.